ayon_api/entity_hub.py

```diff
@@ -2474,6 +2505,7 @@ def from_entity_data(cls, project, entity_hub):
             folder_types=project["folderTypes"],
             task_types=project["taskTypes"],
             name=project["name"],
+            statuses=project["statuses"],
             attribs=project["ownAttrib"],
```

Comment:
@iLLiCiTiT and is this correct? wouldn't it be `project["attrib"]` instead? and why does `ownAttrib` not work for the version entity?

Reply:
Because you set `own_attributes=False`.

Reply:
Well like I say on the code, if I set it to True it errors out...

Reply:
Aaah, I found out that only projects, folders and tasks have `ownAttrib` will make PR marking the argument deprecated in other methods.

Reply:
Haha and does that make sense to pass it on as the "attribs" argument on the creator? What would be the equivalent for that for the other entities?